decidim-admin/app/helpers/decidim/admin/attributes_display_helper.rb

```diff
@@ -31,7 +31,14 @@ module AttributesDisplayHelper
       # attrs - a list of N attributes of the `record`.
       def display_for(record, *attrs)
         attrs.map do |attr|
-          display_label(record, attr) + display_value(record, attr)
+          if record.column_for_attribute(attr).type == :hstore
```

Comment:
Nope, the model doesn't know anything about translations, only the form objects, so I can't inspect that. Anyway, the system allows you to manually display an attribute, so if your attribute is an `hstore` but not a translation one you could customize the display.